source/features/i18n/translations/de.json

```diff
@@ -161,13 +161,22 @@
     "to": "Output-Adressen",
     "id": "Transaktions-ID",
     "includedIn": "Inkludiert in",
+    "metadata": "Metadata",
     "noTransactions": "Keine Transaktionen",
     "pageTitle": "Transaktion | $t(productTitle)",
     "receivedTime": "Erhalten",
     "summary": "Übersicht",
     "totalOutput": "Output Insgesamt",
     "transactionLabel": "Transaktion",
-    "transactionsLabel": "Transaktionen"
+    "transactionsLabel": "Transaktionen",
+    "unmoderated": "!!!You have consented to view unmoderated transaction data.",
+    "unmoderatedConfirmation": "!!!Yes, show unmoderated content",
+    "unmoderatedContent": "!!!Do you want to see unmoderated content?",
+    "unmoderatedEntitiesControl": "!!!This information is provided and hosted by stake pool operators and it is not in control of entities running this website.",
+    "unmoderatedLeave": "!!!Leave this page",
+    "unmoderatedPoolTickerSymbols": "!!!Transaction metadata is not moderated and may contain inappropriate content.",
+    "unmoderatedTransactionData": "!!!Transaction metadata data is not moderated",
+    "unmoderatedWarning": "!!!Warning:"
```

Comment:
What about the German translation? Do we still get this from someone?

Reply:
Added